tests/modules/mcuboot/external_flash/Kconfig.sysbuild

```diff
@@ -7,4 +7,18 @@
 config PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY
 	default y if BOARD_NRF5340DK_NRF5340_CPUAPP || BOARD_NRF52840DK_NRF52840
 
+choice MCUBOOT_MODE
+	default MCUBOOT_MODE_OVERWRITE_ONLY if SOC_SERIES_NRF53X
+endchoice
+
+choice NETCORE
+	default NETCORE_EMPTY if BOARD_NRF5340DK_NRF5340_CPUAPP || BOARD_THINGY53_NRF5340_CPU
+endchoice
+
+config SECURE_BOOT_NETCORE
+	default y if BOARD_NRF5340DK_NRF5340_CPUAPP || BOARD_THINGY53_NRF5340_CPUAPP
+
+config NETCORE_APP_UPDATE
+	default y if BOARD_NRF5340DK_NRF5340_CPUAPP || BOARD_THINGY53_NRF5340_CPUAPP
```

Comment:
Here I think you could just use 1 option which is `SOC_NRF5340_CPUAPP` I would assume this satisfies your requirement to enable these KConfigs.

Reply:
That being said the options you chose better reflects what are the supported boards.